### ISSUE TITLE
NullPointerException, possibly caused by Booster Boots

### ISSUE THREAD

**Issue by JDSMADCLOWN** [BenWoodworth/FastCraftPlus#50]
# Description
Exception doesn't occur when the items needed to craft the boots are not in your inventory.
Booster Boots: https://www.spigotmc.org/resources/booster-boots.21792/

# Debug Output
<details><summary>Show **/fcadmin debug** output</summary>
```
========== FastCraft+ Debug Info ==========
Versions:
FastCraft+ v0.19.4
git-Spigot-1480adb-8b61cc5 (MC: 1.9.2)
1.9.2-R0.1-SNAPSHOT
Java 1.8.0_91
Other Plugins (58):
ArmorPlus 1.10-BETA
AsyncWorldEdit 3.3.7
AsyncWorldEditInjector 3.3.5
Autorank 3.7.3
BetterMcmmoStats 1.2.1
Booster-Boots 1.7.0
BungeeTabListPlus 2.4.4
BuycraftX 10.0
ChestShop 4.0-r5
Citizens 2.0.17-SNAPSHOT (build 1340)
ColorName 2.0.0
CommandNPC 1.7.2
CrackShot 0.98.5
CraftableDisks 3.0.2
CraftBook 3.9-SNAPSHOT
dynmap 2.3-SNAPSHOT-1132
EnderDoors 1.3.0
Essentials TeamCity
EssentialsAntiBuild TeamCity
EssentialsChat TeamCity
EssentialsProtect TeamCity
EssentialsSpawn TeamCity
Factions 2.8.13
FactionsDynmap 2.8.13
FindDeathLocation 1.5.5
GroupManager 2.0 (Phoenix)
HeartDrops 1.5.3
Jobs 3.3.6
LockettePro 2.1.2
Magic 6.1.1
MassiveCore 2.8.13
mcMMO 1.5.05-SNAPSHOT-b133
mcMMOAction 0.9
MCMMOShop 1.0
MobCatcher 3.26
MoreHearts 1.1
Multiverse-Core 2.5-b719
Multiverse-Portals 2.5-b742
MysqlPlayerDataBridge 3.1.1
MythicMobs 2.4.1
PickupMoney 2.3
ProjectKorra 1.8.0 BETA 9
ProtocolLib 4.0.0-SNAPSHOT-b269
pUtils 1.2.1
React 1.6p
RockMC_BendingMenu 1.4.0
Sentry 1.9.0
SilkSpawners 3.7.1
SimpleBungeePortals 1.0
Stats 3.1.0-RC-431
TradeMe 3.4.0.7
Vampire 2.8.13
Vault 1.5.6-b49
VisualWorldEdit 2.1
WeaponsPlus 1.6.1
WorldBorder 1.8.5
WorldEdit 6.1;no_git_id
WorldGuard 6.1
===========================================
```
</details>

# Stack Trace
<details><summary>Show stack trace</summary>
```
[21:45:31] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to FastCraftPlus v0.19.4
org.bukkit.event.EventException
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at org.bukkit.craftbukkit.v1_9_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:231) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.PlayerInteractManager.a(PlayerInteractManager.java:482) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.PlayerConnection.a(PlayerConnection.java:888) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.PacketPlayInUseItem.a(SourceFile:55) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.PacketPlayInUseItem.a(SourceFile:11) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_91]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_91]
	at net.minecraft.server.v1_9_R1.SystemUtils.a(SourceFile:45) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.MinecraftServer.D(MinecraftServer.java:721) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.DedicatedServer.D(DedicatedServer.java:400) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.MinecraftServer.C(MinecraftServer.java:660) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at net.minecraft.server.v1_9_R1.MinecraftServer.run(MinecraftServer.java:559) [spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_91]
Caused by: java.lang.NullPointerException
	at java.util.LinkedList.addAll(Unknown Source) ~[?:1.8.0_91]
	at co.kepler.fastcraftplus.craftgui.buttons.GUIButtonRecipe.getItem(GUIButtonRecipe.java:80) ~[?:?]
	at co.kepler.fastcraftplus.api.gui.GUI.updateLayout(GUI.java:139) ~[?:?]
	at co.kepler.fastcraftplus.craftgui.GUIFastCraft.updateLayout(GUIFastCraft.java:96) ~[?:?]
	at co.kepler.fastcraftplus.craftgui.GUIFastCraft.<init>(GUIFastCraft.java:58) ~[?:?]
	at co.kepler.fastcraftplus.craftgui.PlayerManager.onPlayerInteract(PlayerManager.java:46) ~[?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_91]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_91]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_91]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_91]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-1.9.2.jar:git-Spigot-1480adb-8b61cc5]
	... 17 more
```
</details>

**Comment by BenWoodworth** [comment]
https://github.com/BenWoodworth/FastCraftPlus/blob/v0.19.4/src/main/java/co/kepler/fastcraftplus/craftgui/buttons/GUIButtonRecipe.java#L80

**Comment by BenWoodworth** [comment]
Should be fixed in v0.20.4
